pkg/cmd/server/admin/create_nodeconfig.go

```diff
@@ -395,7 +395,17 @@ func (o CreateNodeConfigOptions) MakeNodeConfig(serverCertFile, serverKeyFile, n
 		return err
 	}
 
-	content, err := latestconfigapi.WriteYAML(config)
+	// Roundtrip the config to v1 and back to ensure proper defaults are set.
```

Comment:
@deads2k with @liggitt's help we discovered we need to add this to get defaulting of the NodeConfig. He suggested I let you know as we should probably unify this and BuildSerializeableNodeConfig